checker/src/main/java/org/checkerframework/checker/nullness/junit-assertions.astub

```diff
@@ -0,0 +1,3453 @@
+/*
+ * Copyright 2015-2021 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * https://www.eclipse.org/legal/epl-v20.html
```

Comment:
Do we need to update our licenses file and the licenses of the Maven artifact to include this?

Reply:
We should probably recreate the file from scratch, minimally, using the public API.  Then licensing is not an issue.